.github/workflows/builds.yml

```diff
@@ -11,18 +11,23 @@ jobs:
     steps:
       - name: Clone the repository
         uses: actions/checkout@v2
-      - run: make release
-      - run: ls -lH mainnet-release.wasm
+      - name: Install cargo-make
```

Comment:
There's actually no need to do this. Since our runners are persistent, I will just update the runner image, and `cargo-make` will be forever available.

Reply:
I've [installed cargo-make to runners](https://github.com/aurora-is-near/devops-stuff/commit/9e8b7c9a299c1b833ece28f93c3afe78af05d3dc?diff=unified), so you can remove this step from all workflows

Reply:
Ok, I'll remove it.

Reply:
One place still left)